joomla/joomla-cms
# Disable show_email_copy per default for new installations

### Summary of Changes

Disable show_email_copy per default for new installations. cc @SniperSister 

### Testing Instructions

- Take a new install from https://github.com/zero-24/joomla-cms/archive/copymail.zip
- Check the com_contact options and that "show_email_copy" is disabled
- check that per default a new form does not have show_email_copy enabled

### Expected result

per default a new form on a new install does not have show_email_copy enabled

### Actual result

per default a new form have show_email_copy enabled
